compiler/hash-lower/src/build/matches/test.rs

```diff
@@ -15,7 +15,7 @@ use hash_ir::{
 };
 use hash_reporting::macros::panic_on_span;
 use hash_source::constant::{IntConstant, IntConstantValue, InternedInt};
-use hash_storage::store::statics::StoreId;
+use hash_storage::store::statics::CoreStoreId;
```

Comment:
There is alot PR noise here, maybe this should be in a different PR?

Reply:
You mean the rename? I also was thinking of undoing this change tbh, instead having `StoreId -> SingleStoreId` and `CoreStoreId -> StoreId` which should restore most of the old names...

Reply:
Yeah, I think keeping old names would be better. In terms of users of the stores, the `CoreStoreId` name seems a bit strange